src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -285,6 +291,18 @@ private static void createBranches(List<LfBus> lfBuses, LfNetwork lfNetwork, Loa
             addBranch(lfNetwork, LfDanglingLineBranch.create(danglingLine, lfNetwork, lfBus1, lfBus2), report);
         }
 
+        for (TieLine tieLine : loadingContext.tieLines) {
+            LfTieLineBus lfBoundaryBus = new LfTieLineBus(lfNetwork, tieLine);
+            lfNetwork.addBus(lfBoundaryBus);
+            lfBuses.add(lfBoundaryBus);
+            LfBus lfBus1 = getLfBus(tieLine.getTerminal1(), lfNetwork, parameters.isBreakers());
+            LfBus lfBus2 = getLfBus(tieLine.getTerminal2(), lfNetwork, parameters.isBreakers());
+            // the half line at side 1 has its network terminal at terminal 1.
+            addBranch(lfNetwork, LfTieLineBranch.create(tieLine, Branch.Side.ONE, lfNetwork, lfBus1, lfBoundaryBus), report);
```

Comment:
I have some questions about this: when we simulate a contingency on a tie line, how does it works? I think with that PR, it doesn't work anymore. The flow transfer and the BranchResult have to be implemented and is going to be shared by 2 LfBranches, no? Secondly, how does it works in sensi when the monitored branch is a tie line?